lib/bank_accounts/checking_account.rb

```diff
@@ -0,0 +1,46 @@
+module BankAccounts
+  require "pry"
+
+  class CheckingAccount < Account
+    attr_reader :checks_used
+
+    def initialize (id, initial_balance)
+     super
+      @checks_used = 0
+    end
+
+    def withdraw(amount)
+      # adds $1 transaction fee
+      # returns error message if balance drops below zero
+      amount += 1
```

Comment:
This is a smart way to update the `amount` var so you can re-use the super method